apps/shell/tash_main.c

```diff
@@ -423,8 +424,15 @@ int tash_execute_cmdline(char *buff)
 					*(buff + 1) = ASCII_NUL;
 					break;
 
+				case ASCII_RSQUOTE:
```

Comment:
How about below for consistency with `IN_APOSTROPHY` , `case IN_APOSTROPHY:`
`case ASCII_APOSTROPHY:`